pantheon-db-dump

```diff
@@ -2,6 +2,9 @@
 # pantheon-db-dump
 # Dump a DB from a Pantheon environment.
 
+# Check if mysql is installed locally.
+command -v mysql >/dev/null 2>&1 || { echo >&2 "FAILURE. MySQL is required. Please install mysql locally and run the command again."; exit 1; }
```

Comment:
We don't use the `mysql` command in this script.  Instead we should check for `mysqldump`.

Reply:
OK I changed this. I used `mysql` initially because I didn't think `mysqldump` existed without `mysql`. But if that is the command that is actually being used, probably best to test against that command?